src/commands/alias/set.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { SfCommand } from '@salesforce/sf-plugins-core';
+import { StateAggregator, Messages } from '@salesforce/core';
+import validateArgs from '../../shared/validate-args';
+import { AliasResults } from '../../types';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.load('@salesforce/plugin-settings', 'alias.set', [
+  'summary',
+  'description',
+  'examples',
+  'error.ArgumentsRequired',
+  'error.DuplicateArgument',
+  'error.InvalidArgumentFormat',
+  'error.ValueRequired',
+]);
+
+export default class AliasSet extends SfCommand<AliasResults> {
+  public static summary = messages.getMessage('summary');
+  public static description = messages.getMessage('description');
+  public static examples = messages.getMessages('examples');
+
+  // This allows varargs
+  public static readonly strict = false;
```

Comment:
Unrelated to this PR: This causes issues with flag spelling mistakes. Typing `bin/dev alias set --hekp` takes "--hekp" as an arg and not an unknown flag.

Reply:
Log a bug

Reply:
https://github.com/oclif/core/issues/526

Reply:
https://github.com/oclif/core/issues/526